sw/device/lib/dif/dif_csrng.h

```diff
@@ -263,17 +373,17 @@ dif_result_t dif_csrng_update(const dif_csrng_t *csrng,
  *
  * @param csrng A CSRNG handle.
  * @param len Number of uint32_t words to generate.
- *
  */
 OT_WARN_UNUSED_RESULT
 dif_result_t dif_csrng_generate_start(const dif_csrng_t *csrng, size_t len);
 
 /**
  * Reads the output of the last CSRNG generate call.
  *
- * This function blocks until all data is read from the CSRNG. The caller is
- * responsible for making sure that the `len` parameter matches the length set
- * in the `dif_csrng_generate_start()` call.
+ * This function reads `len` words out of the CSRNG. This function should be
+ * called repeatedly until the number of words requested in
+ * `dif_csrng_generate_start()` is exhausted. This function will block until
+ * `len` words are read.
  *
  * `dif_csrng_get_output_status()` can be called before this function to ensure
  * there is data availble in the CSRNG output buffer.
```

Comment:
Would you mind fixing this typo?
```suggestion
 * there is data available in the CSRNG output buffer.
```
 * there is data availble in the CSRNG output buffer.